### PR TITLE
README to explain exclusion files

### LINKED ISSUE #1646: cloud-opensource-java/dependencies/README.md should discuss exclusion files
https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/dependencies/README.md

### PR DESCRIPTION
**Pull request by suztomo**
Closes #1646 .

### CODE REVIEW

File: dependencies/README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 
-Linkage Checker is a tool that finds [linkage errors](
+This is a developer note for Linkage Checker, a tool to find [linkage errors](
 ../library-best-practices/glossary.md#static-linkage-error) on a class
 path. It scans the class files in the class path for references to other
 classes and reports any reference that cannot be satisfied.
 
+#### User Documentation
+
+We provide Linkage Checker with integrations for Maven and Gradle. Here are the user documentation:
```
Comment (suztomo) [inline dependencies/README.md:10]: Clarifying this is not for user documentation.

File: dependencies/README.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ Linkage Checker is a tool that finds [linkage errors](
...
```
Comment (elharo) [inline dependencies/README.md:10]: Linkage Checker can be used from Maven or Gradle.

File: dependencies/README.md [hunk 3]
```diff
@@ -68,3 +77,14 @@ Entry point classes are different for the input of checks:
   - **Check for a list of Maven coordinates**: classes in the Maven artifacts
   - **Check for a list of class and jar files**: all classes in the input are entry points.
     This means that every linkage error is reachable.
+
+### Exclusion Files
+
+Users can specify an exclusion file to filter out known linkage errors that we know are not harmful.
```
Comment (elharo) [inline dependencies/README.md:83]: delete "that we know are not harmful".
In general, Google style avoids "we" (first person plural)

File: dependencies/README.md (hunk 2 above)
Comment (suztomo) [reply dependencies/README.md]: Updated.

File: dependencies/README.md (hunk 3 above)
Comment (suztomo) [reply dependencies/README.md]: Updated

**Review by suztomo — COMMENTED** [review]
@elharo PTAL.

**Review by elharo — APPROVED** [review]

**Pull request merged**